variables.tf

```diff
@@ -29,6 +29,12 @@ variable "iam_role_name" {
   description = "The IAM role name. Required to match with iam_role_arn if use_existing_iam_role is set to `true`"
 }
 
+variable "permission_boundary_arn" {
+  type        = string
+  default     = null
```

Comment:
Same question as other changes

Should default be null or empty string?

Reply:
Null, here is why:
Null a value that represents absence or omission. If you set an argument of a resource to null, Terraform behaves as though you had completely omitted it — it will use the argument's default value if it has one, or raise an error if the argument is mandatory. null is most useful in conditional expressions, so you can dynamically omit an argument if a condition isn't met.
https://developer.hashicorp.com/terraform/language/expressions/types#null